p2plending/frontend/src/app/terms/Terms.js

```diff
@@ -0,0 +1,145 @@
+              territory (i.e. region or country) without turning off data
+              roaming. If you are not the bill payer for the device on
+              which you're using the app, please be aware that we assume
+              that you have received permission from the bill payer for
+              using the app.
+            </p> 
+            <p>
+              Along the same lines, Open San Diego Org cannot always
+              take responsibility for the way you use the app i.e. You
+              need to make sure that your device stays charged and if it
+              runs out of battery and you can't turn it on to avail the
+              Service, Open San Diego Org cannot accept
+              responsibility.
+            </p> 
+            <p>
+              With respect to Open San Diego Org's responsibility for
+              your use of the app, when you're using the app, it's
+              important to bear in mind that although we endeavour to
+              ensure that it is updated and correct at all times, we may
+              rely on third parties to provide information to us so that
+              we can make it available to you.
+              Open San Diego Org accepts no liability for any loss,
+              direct or indirect, you experience as a result of relying
+              wholly on this functionality of the app.
+            </p> 
+            <p>
+              At some point, we may wish to update the app.
+              Open San Diego Org does not promise that
+              it will always update the app so that it is relevant to
+              and/or works with your current hardware and software. 
+              However, you promise to always
+              accept updates to the application when offered to you, We
+              may also wish to stop providing the app, and may terminate
+              use of it at any time without giving notice of termination
+              to you. Unless we tell you otherwise, upon any termination,
+              (a) the rights and licenses granted to you in these terms
+              will end; (b) you must stop using the app, and (if needed)
+              delete it from your device.
+            </p> 
+            <p><strong>Changes to This Terms and Conditions</strong></p> <p>
+              We may update our Terms and Conditions
+              from time to time. Thus, you are advised to review this page
+              periodically for any changes. We will
+              notify you of any changes by posting the new Terms and
+              Conditions on this page. These changes are effective
+              immediately after they are posted on this page.
+            </p> 
+            <p><strong>Contact Us</strong></p> <p>
+              If you have any questions or suggestions about
+              our Terms and Conditions, do not hesitate to
+              contact us at https://opensandiego.org/
```

Comment:
Hyperlink the "https://opensandiego.org" text to link to our website

Reply:
done